ProcessMaker/processmaker
# FOUR-13067: Add icons in Process Launchpad

## Issue & Reproduction Steps
Add this list of icons in launchpad icons:

[Winter24](https://www.figma.com/file/bI3YLt9n2771ReExAQIggs/Winter24?type=design&node-id=7422-8527&mode=design)

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
1. Go to Launchpad
2. Open a process
3. Go to Launchpad Settings
4. See the new list of icons are available

## Related Tickets & Packages
- [FOUR-13067](https://processmaker.atlassian.net/browse/FOUR-13067)

## Code Review Checklist


ci:next

[FOUR-13067]: https://processmaker.atlassian.net/browse/FOUR-13067?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
